### ISSUE TITLE
Seems broken on V 2.7.1

### ISSUE THREAD

**Issue by philipboomy** [peda/statamic-glide-vanity-url#1]
Cleared stache and cache but I am still getting this url:

`<img src="/img/id/main::img/our-team.jpg/our-team.jpg?q=80&amp;s=09f6c431cb0f249f141434001051198a">`

This is how it looks without the add-on:
`<img src="/img/asset/bWFpbi9pbWcvb3VyLXRlYW0uanBn?q=80&amp;s=e895d2bccfa4e6f839e736d70e0cdc37">`

**Comment by peda** [comment]
Yes, it is broken - Statamic did change the image URL format (in 2.5.0 release) - even though it was just a "minor" version it did contain many breaking changes and broke quite a few addons (including this one).

I've tried keeping up with the changes right afterwards but they kept changing so many classes every related to assets and Glide, that I did decide not to update it for now. It looks like this is now stabilizing a bit since the 2.7.0 release. Maybe I'll have a look at it again and see how this can be updated to the latest version.
